articles/communication-services/quickstarts/email/send-email-smtp/smtp-authentication.md

```diff
@@ -28,7 +28,7 @@ Application developers who build apps that send email using the SMTP protocol ne
 
 ### Creating a custom email role for the Entra application
 
-The Entra application must be assigned a role with both the **Microsoft.Communication/CommunicationServices/Read** and the **Microsoft.Communication/EmailServices/write** permissions on the Azure Communication Service Resource. This can be done either by using the **Contributor** role, or by creating a **custom role**. Follow these steps to create a custom role by cloning an existing role.
+The Entra application must be assigned a role with both the **Microsoft.Communication/CommunicationServices/Read**, **Microsoft.Communication/CommunicationServices/Write**, and the **Microsoft.Communication/EmailServices/write** permissions on the Azure Communication Service Resource. This can be done either by using the **Contributor** role, or by creating a **custom role**. Follow these steps to create a custom role by cloning an existing role.
```

Comment:
Suggestion: Looking at the preview of the text, the permissions don't seem to stand out very well despite the fact that they're bold. It might be worth putting them between ticks instead i.e. `Microsoft.Communication/CommunicationServices/Read`.